contrib/builder/rpm/fedora-22/Dockerfile

```diff
@@ -7,7 +7,7 @@ FROM fedora:22
 RUN dnf install -y @development-tools fedora-packager
 RUN dnf install -y btrfs-progs-devel device-mapper-devel glibc-static libseccomp-devel libselinux-devel libtool-ltdl-devel selinux-policy selinux-policy-devel sqlite-devel tar
 
-ENV SECCOMP_VERSION v2.2.3
+ENV SECCOMP_VERSION 2.2.3
```

Comment:
Yeah, this is good -- she updated the lines below to match the top-level `Dockerfile` too, which was the issue. :+1:

(This could actually be a separate PR and it'd be merged already. :smile:)